### PR TITLE
CI: Add windows.

### PR DESCRIPTION
**Pull request by ryan-roemer**
## Work
- Add more badges!!!
- Add windows to CI run via appveyor.
- Add node11
- Switch pkg.json quotes to double for win.

## Notes
We uncovered some minor stuff, but pretty easy!

I originally tried this on Travis windows, but it's **so, so slow** 🐢 

Screen Shot 2019-05-14 at 8 16 26 PM

Lets compare those times to appveyor 😂 

Screen Shot 2019-05-14 at 9 03 47 PM

### CODE REVIEW

File: utils/format-modules.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function _formatFileName(mod) {
 
   // Source file.
   if (!baseName) {
-    return `{green-fg}./${relative(process.cwd(), resolve(fileName))}{/}`;
+    return `{green-fg}.${sep}${relative(process.cwd(), resolve(fileName))}{/}`;
```
Comment (ryan-roemer) [inline utils/format-modules.js]: Ugh. This makes this PR a (minor, minor, ) minor semver release....

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "endOfLine": "auto"
```
Comment (ryan-roemer) [inline .prettierrc]: Windows prettier will change linefeed in a way that blows up without this.

Comment (parkerziegler) [reply .prettierrc]: Yeah I was seeing that here: https://github.com/prettier/prettier/pull/5327, really good catch!

File: README.md [hunk 3]
```diff
@@ -159,3 +161,9 @@ Error output deeply inspired by: [https://github.com/facebookincubator/create-re
 **Active:** Formidable is actively working on this project, and we expect to continue for work for the foreseeable future. Bug reports, feature requests and pull requests are welcome.
 
 [maintenance-image]: https://img.shields.io/badge/maintenance-active-green.svg
+[npm_img]: https://img.shields.io/npm/v/webpack-dashboard.svg?style=flat
```
Comment (parkerziegler) [inline README.md]: Love this style of Markdown badge and site referencing :chefs-kiss:

File: package.json [hunk 4]
```diff
@@ -9,15 +9,15 @@
   },
   "types": "index.d.ts",
   "scripts": {
-    "test": "mocha 'test/**/*.spec.js'",
-    "test-cov": "nyc mocha 'test/**/*.spec.js'",
+    "test": "mocha \"test/**/*.spec.js\"",
```
Comment (parkerziegler) [inline package.json]: Is this also to make Windows happy?

File: utils/format-modules.js (hunk 1 above)
Comment (parkerziegler) [reply utils/format-modules.js]: No worries, I'll take care of it after merge!

File: package.json (hunk 4 above)
Comment (ryan-roemer) [reply package.json]: Yeah, some terminals seem to blow up with single quotes. But that may be me or my VM? In any case, this works 😛

**Pull request merged**